packages/client/e2e/_utils/mpe-e2e-utils.ts

```diff
@@ -320,6 +321,31 @@ export async function searchAndJoinMpeRoomFromMpeRoomsSearchEngine({
     }
 }
 
+export async function openMpeSettingsModal({
+    page,
+}: {
+    page: Page;
+}): Promise<void> {
+    const openSettingsButton = page.locator(
+        `css=[data-testid="mpe-open-settings"]`,
+    );
+    expect(openSettingsButton).toBeTruthy();
+    await expect(openSettingsButton).not.toBeDisabled();
+
+    await openSettingsButton.click();
+
+    await waitFor(() => {
+        const exportButton = page.locator(
+            `css=[data-testid="export-mpe-to-mtv-button"]`,
+        );
+        expect(exportButton).toBeTruthy();
+        const leaveRoomButton = page.locator(
+            `css=[data-testid="leave-mpe-room-button"]`,
+        );
+        expect(leaveRoomButton).toBeTruthy();
```

Comment:
`toBeTruthy` assertion will do nothing in E2E tests. Actually, it will assert that `leaveRoomButton` is not falsy. Here, `leaveRoomButton` is an object of type `Locator`, and not the DOM elements pointed by the selector. We should be using `toBeVisible`, or any other assertion that will resolve the `Locator` into real elements.

Reply:
I continued to read all the files, and I understand why you wrote that. As our E2E tests and our integration tests have quite the same API, we can easily get lost between each other 😄

Reply:
You're right mistake there, fixing it